### ISSUE TITLE
Regarding the issue of training process

### ISSUE THREAD

**Issue by tianmengge** [sascha-kirch/samba-mixer#10]
Good sir, I appreciate your inspiring work.

question 1 : you utilize the test set as a validation set. Will this lead to data leakage？

question 2 : If so, is eval.py still useful? Isn't the result the last round of training
